src/bootstrap-table.js

```diff
@@ -2002,26 +2006,37 @@
         }
 
         $.each(this.columns, function (i, column) {
-            var falign = '', // footer align style
-                style = '',
+            var key,
+                falign = '', // footer align style
+                valign = '',
+                csses = [],
+                style = {},
                 class_ = sprintf(' class="%s"', column['class']);
-
+ 
```

Comment:
Oh fuck off you daft cunt.

On Saturday, 27 February 2016, Dustin U. <notifications@github.com> wrote:

> In src/bootstrap-table.js
> <https://github.com/wenzhixin/bootstrap-table/pull/1887#discussion_r54339461>
> :
>
> >                  class_ = sprintf(' class="%s"', column['class']);
> > -
> > +
>
> bad whitespaces [image: :-1:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wenzhixin/bootstrap-table/pull/1887/files#r54339461>.
>